src/Compilers/Core/Portable/CommandLine/ErrorLogger.cs

```diff
@@ -15,28 +15,31 @@ namespace Microsoft.CodeAnalysis
     /// <summary>
     /// Used for logging all compiler diagnostics into a given <see cref="Stream"/>.
     /// This logger is responsible for closing the given stream on <see cref="Dispose"/>.
+    /// It is incorrect to use the logger concurrently from multiple threads.
+    ///
     /// The log format is SARIF (Static Analysis Results Interchange Format)
+    /// https://sarifweb.azurewebsites.net
     /// https://github.com/sarif-standard/sarif-spec
     /// </summary>
     internal partial class ErrorLogger : IDisposable
     {
-        // Internal for testing purposes.
-        internal const string OutputFormatVersion = "0.4";
-
         private readonly JsonWriter _writer;
+        private readonly DiagnosticDescriptorSet _descriptors;
+        private readonly CultureInfo _culture;
 
-        public ErrorLogger(Stream stream, string toolName, string toolFileVersion, Version toolAssemblyVersion)
+        public ErrorLogger(Stream stream, string toolName, string toolFileVersion, Version toolAssemblyVersion, CultureInfo culture)
```

Comment:
I'd probably just add a separate constructor - even though this is not public API, I know of a bunch of tools that use reflection to instantiate an error logger. We can avoid breaking them..